core/data/src/main/java/uk/gov/gchq/gaffer/data/elementdefinition/view/View.java

```diff
@@ -197,6 +198,14 @@ public void expandGlobalDefinitions() {
         }
     }
 
+    public void setHasBeenMigrated(final boolean hasBeenMigrated) {
```

Comment:
Would it be possible to avoid this? Perhaps it would be neater to have a field:
private Map<String, String> config;

where you can just set arbitrary config in it. This would avoid haven't to add specific logic for migration into the core code.